task002/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/karamel29/go-courses/task002/fibonacci"
+)
+
+func main() {
+	fmt.Println("Hi")
+	defer fmt.Println("Mission accomplished")
+	fibonacci.Printer(5)
```

Comment:
Although the original Fibonacci sequence did start with 1 and 2 nowadays it is no longer considered correct. Now it is: 0, 1, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987, 1597. How do you change the function to get the initial values (0, 1)?